ecs/ecs.go

```diff
@@ -97,15 +100,15 @@ func (c *Client) GetResourceGraph() (ResourceNode, error) {
 			}
 		}
 
-		slaveNode.CPUTotal = usedCPU
-		slaveNode.MemoryTotal = usedMem
-		slaveNode.CPU = usedCPU - remainingCPU
-		slaveNode.Memory = usedMem - remainingMem
+		slaveNode.CPUTotal = usableCPU
+		slaveNode.MemoryTotal = usableMem
+		slaveNode.CPU = usableCPU - remainingCPU
+		slaveNode.SoftMemory = usableMem - remainingMem
```

Comment:
re: "used" => "usable" ... was this named incorrectly before?

Reply:
Yes, I believe so.